.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.0
```

Comment:
I'm pinning the version of ruby here. ELEC-154 tracks an upstream issue that once merged, will allow us to pin chef-solo versions without resorting to a hack.